.github/workflows/LABEL_COMMENTS.yml

```diff
@@ -0,0 +1,92 @@
+name: Add comment
+on:
+  issues:
+    types:
+      -labeled
+
+jobs:
+  status-backlog:
+    if: github.event.label.name == "status:backlog"
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add "status:backlog" comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thank you for your contribution. Your support of Quire is greatly appreciated. We are reviewing your suspected bug and will be in touch soon.
```

Comment:
This is the message for "triage needed." The message for "backlog" is: "Your bug is important to us, but we can’t get to it quite yet. Stay tuned. We will confirm when your bug has been selected for development.” I also think we can remove the "stay tuned." It seems overly wordy.